aws-datastore/src/main/java/com/amplifyframework/datastore/AWSDataStorePlugin.java

```diff
@@ -93,14 +95,16 @@ private AWSDataStorePlugin(
         this.sqliteStorageAdapter = SQLiteStorageAdapter.forModels(modelSchemaRegistry, modelProvider);
         this.categoryInitializationsPending = new CountDownLatch(1);
         this.authModeStrategy = AuthModeStrategyType.DEFAULT;
+        this.isSyncRetryEnabled = false;
```

Comment:
Wouldn't you want to set this to `userProvidedConfiguration.doSyncRetry`?

Reply:
Updated.